src/APIM_ARMTemplate/apimtemplate/TemplateCreators/APITemplateCreator.cs

```diff
@@ -16,44 +16,23 @@ public async Task<APITemplate> CreateAPITemplateAsync(OpenApiDocument doc, CLICr
             // create api schema with properties
             APITemplate apiSchema = new APITemplate()
             {
-                name = doc.Info.Title,
                 type = "Microsoft.ApiManagement/service/apis",
                 apiVersion = "2018-06-01-preview",
                 properties = new APITemplateProperties()
```

Comment:
>APITemplateProperties [](start = 33, length = 21)

another thing to note here is that importing a Swagger file will be a two step process
1. As discussed the first Template will just create a API facade (in the correct ApiRevision / ApiVersionSet) with dummy serviceUrl, correct Path and dummy 
2. The second template will call the swagger import format with `contentFormat`  set to `swagger-json` 

Here you are merging both the steps together, which will not work.